source/js/next-boot.js

```diff
@@ -2,8 +2,6 @@
 
 $(document).ready(function() {
 
-  $(document).trigger('bootstrap:before');
```

Comment:
Somebody help confirm this `trigger` triggers nothing...

Reply:
I think it would be better to keep it as it allows users to listen to NexT scripts in their own external js scripts

Reply:
These events are not mentioned in the official documentation... So I think users should inject code by themself...

Reply:
See: https://github.com/iissnan/hexo-theme-next/commit/029a4296bdac15812234d16b5c43adc6d3cb195f